docs/en/enterprise-edition/content-collections/application-security/ides/connect-vscode.adoc

```diff
@@ -40,9 +40,9 @@ NOTE: The plugin automatically invokes the latest version of Prisma Cloud Code S
 
 .. For *GRBAC*, provide the following permissions.
 +
-* *Policies*: 'View' permissions
-* *Projects*: 'View' permissions
-* *Repositories*: ('View' and 'Create') OR ('View' and 'Update') permissions
+* *Policies* -> *Policies*: 'View' permissions
+* *Application Security* -> *Projects*: 'View' permissions
```

Comment:
What about all other features besides Application Security? Should we have a note?

Reply:
The first is the header
<img width="865" alt="Screenshot 2024-10-27 at 10 44 12 PM" src="https://github.com/user-attachments/assets/0515d75a-0620-445f-b68a-817201f66889">